scout/constants/variants_export.py

```diff
@@ -14,6 +14,16 @@
 ]
 
```

Comment:
Not your fault, but have you seen this?

<img width="792" alt="image" src="https://github.com/Clinical-Genomics/scout/assets/28093618/7542ea41-5d4c-45b2-b16d-ed8b826efc68">

🤣 🤣 🤣

Reply:
Yes... Im part guilty now as well, but it's mostly because of a very looong sequence of "if this value exists, do that". While not so DRY perhaps it's also not really complex. We could break it up in one function per variant category to "fool" the counter, but I dunno..

Reply:
Let's do it in a dedicated PR!

Reply:
By reading this PR I've realised that having a dedicated PR for the export would give that code more justice.. Sorry!